SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -205,15 +207,39 @@ public class SwiftyTesseract {
     }
     
     // create unique file path
-    let filepath = try processPDF(images: images)
-    
+    let filepath = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(UUID().uuidString)
+    try processPDF(images: images, at: filepath)
+
     // get data from pdf and remove file
     let data = try Data(contentsOf: filepath)
     try FileManager.default.removeItem(at: filepath)
     
     return data
   }
-  
+
+  /// Takes an array UIImages and saves a PDF at the given path.
+  /// If using PDFKit introduced in iOS 11, this will produce a valid
+  /// PDF Document.
+  ///
+  /// - Parameters:
+  ///   - images: Array of UIImages to perform OCR on
+  ///   - path: Path where the PDF document should be saved.
+  /// - Throws: SwiftyTesseractError
+  public func createPDF(from images: [UIImage], at path: URL) throws {
```

Comment:
Based on the prior two comments, I think that this should actually return a URL. If the URL passed in doesn't have the `.pdf` file extension, it will probably cause quite a bit of confusion because the user of the library is going to expect that they would be able to access the URL they passed in without modification. We could annotate this with @discardableResult so we don't force anyone to do a `_ = createPDF(from:at)` to silence a warning.